src/components/Disputes/DisputeEvidences.js

```diff
@@ -154,16 +155,30 @@ const EvidenceContent = React.memo(function EvidenceContent({
   )
 })
 
-function getSubmitterLabel(submitter, dispute) {
-  if (addressesEqual(submitter, dispute.defendant)) {
+function getSubmitterLabel(submitter, defendant, plaintiff) {
+  if (addressesEqual(submitter, defendant)) {
     return 'Defendant'
   }
 
-  if (addressesEqual(submitter, dispute.plaintiff)) {
+  if (addressesEqual(submitter, plaintiff)) {
     return 'Plaintiff'
   }
 
   return ''
 }
 
-export default DisputeEvidences
+export default ({ dispute, evidences }) => {
```

Comment:
Do we need to separate the `useEvidences()` out?

(Otherwise, let's use the function form and add a name to this component).

Reply:
Yes we do since `dispute` and `evidences` are always a new object (due to polling) ? 

What we do here is to ensure we are always providing the same data to `DisputeEvidences` if it has not changed.